execute/plugin.go

```diff
@@ -56,6 +56,10 @@ type Plugin struct {
 	contractsInitialized bool
 }
 
+func (p *Plugin) RreportingCfg() ocr3types.ReportingPluginConfig {
```

Comment:
Temporary, gonna remove that as followup once I extract the shared parts of tests